### DISCUSSION TITLE
PS4 Cheats Manager v1.2.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#109] [Announcements]
### Added

- MC4 format decryption
  - Parse and display codes from MC4 files
  - Export decrypted MC4 files to `xml` format
  - List invalid MC4 files
- Tag cheat files by format (`JSON`, `SHN`, `MC4`)
- Add option to delete all local cheats, patches, plugins
- Network proxy support (based on system settings)

### Changed

- Updated UI assets

### Fixed

- Game filtering by Title ID

### Misc

- Includes latest cheat+patch pack (2023-12-30)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/PS4CheatsManager/releases/tag/v1.2.0'>PS4 Cheats Manager v1.2.0</a>.</em>

**Comment by ApexTam001** [comment]
I can't seem to download it. Please send it to my email samueltam575@gmail.com.. please

**Comment by Alfre8o** [comment]
Xd
